subsys/net/ip/utils.c

```diff
@@ -518,3 +518,230 @@ bool net_header_fits(struct net_pkt *pkt, u8_t *hdr, size_t hdr_size)
 
 	return false;
 }
+
+#if defined(CONFIG_NET_IPV6) || defined(CONFIG_NET_IPV4)
+static bool convert_port(const char *buf, u16_t *port)
+{
+	char *endptr;
+	int tmp;
```

Comment:
While `int` and `long` are the same size, it's better to use the same return type for `strtol()` here: `long int`.